pipeline/installs/multi-user/istio-authorization-config.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: ml-pipeline-ui
+  namespace: kubeflow
+spec:
+  rules:
+  - services:
+    - ml-pipeline-ui.kubeflow.svc.cluster.local
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
```

Comment:
Note service role is deprecated for istio, but they still work in istio 1.4.x.

I'd prefer leaving this here for a while and migrate when all platforms upgrade to later istio versions.